runtime/oti/bcverify_api.h

```diff
@@ -163,7 +163,7 @@ bcvIsInitOrClinit (J9CfrConstantPoolInfo * info);
 * @return UDATA
 */
 UDATA
-j9bcv_checkClassLoadingConstraintsForSignature (J9VMThread* vmThread, J9ClassLoader* loader1, J9ClassLoader* loader2, J9UTF8* sig1, J9UTF8* sig2);
+j9bcv_checkClassLoadingConstraintsForSignature (J9VMThread *vmThread, J9ClassLoader *loader1, J9ClassLoader *loader2, J9UTF8 *sig1, J9UTF8 *sig2, UDATA copySig1);
```

Comment:
As the code is passing in `TRUE/FALSE`, using `BOOLEAN` instead of `UDATA` makes more sense